gui/quickfort.lua

```diff
@@ -0,0 +1,14 @@
+-- in-game dialog interface for the quickfort script
+--[====[
+gui/quickfort
+=============
+In-game dialog interface for the `quickfort` script.
+]====]
+
+local args = {...}
+
+if #args == 0 then
+    dfhack.run_command('quickfort', 'gui')
```

Comment:
I would probably go with `run_script` for Lua scripts (which reminds me that I need to finish https://github.com/DFHack/dfhack/pull/1717 at some point...). Likely a negligible performance difference in this case, though.